commands/info/__tests__/info-command.test.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+jest.mock("envinfo");
+
+const envinfo = require("envinfo");
+
+envinfo.run.mockResolvedValue("MOCK_ENVINFO");
+
+// helper
+const output = require("@lerna/output");
+
+// file under test
+const lernaInfo = require("@lerna-test/command-runner")(require("../command"));
+
+it("outputs result of envinfo()", async () => {
+  // project fixture is irrelevant, no actual changes are made
```

Comment:
Actually, it does matter, as evidenced by the test failures. You can use the `lifecycle` fixture, it's small enough.

Reply:
(That, or the `process.cwd()` below needs to be a more explicit path, like `path.resolve(__dirname, '../../..')`)